swirl/connectors/discord.py

```diff
@@ -0,0 +1,66 @@
+import asyncio
+
+import discord
+from django.conf import settings
+
+from swirl.connectors.connector import Connector
+from swirl.connectors.mappings import *
+
+logger = get_task_logger(__name__)
+path.append(swirl_setdir())
+environ.setdefault('DJANGO_SETTINGS_MODULE', 'swirl_server.settings')
+django.setup()
+
+
+class Discord(Connector):
+    type = "Discord"
+
+    def __init__(self, provider_id, search_id, update, request_id=''):
+        self.client = discord.Client(intents=discord.Intents.all())
+        super().__init__(provider_id, search_id, update, request_id)
+        self.client.event(self.on_ready)
+
+    async def on_ready(self):
+        await self.client.wait_until_ready()
+        await asyncio.sleep(2)
+        #replace your channel id here
+        channel_id = 718114919056146528
+        target_channel = self.client.get_channel(channel_id)
+        if target_channel is None:
+            print('Channel not found')
```

Comment:
Hello, for output, you are using both print and logging. Maybe it would be better to use logging for both of them?

Reply:
Hello, that is only a check for debugging, but yes will be changing it once the code is running .

Reply:
Agree